packages/reporters/liquidation-reporter/excel-writer.ts

```diff
@@ -0,0 +1,116 @@
+// This script takes in info from the liquidation reporter and builds an excel spreadsheet containing condensed information.
+
+const xl = require("excel4node");
+
+export function createExcelSheetFromLiquidationDrawDownData(drawDownData: any) {
+  // Create a new instance of a Workbook class
+  const wb = new xl.Workbook();
+
+  const boldStyle = wb.createStyle({ font: { bold: true } });
+
+  Object.keys(drawDownData).forEach((collateralAddress: string) => {
+    const collateralData = drawDownData[collateralAddress];
+    const sheet = wb.addWorksheet(collateralData.collateralSymbol);
+
+    // Heading
+    sheet
+      .cell(1, 1)
+      .string(`Collateral: ${collateralData.collateralSymbol}`)
+      .style({ font: { size: 16, bold: true } });
+
+    // Collateral info
+    sheet
+      .cell(2, 1)
+      .string("Collateral Price(USD)")
+      .style(boldStyle);
+    sheet.cell(2, 2).number(Number(collateralData.collateralPriceInUsd));
+    sheet
+      .cell(3, 1)
+      .string("Collateral TVL(USD)")
+      .style(boldStyle);
+    sheet.cell(3, 2).number(Number(Number(collateralData.collateralValueInUsd).toFixed(2)));
+    sheet
+      .cell(4, 1)
+      .string("Collateral Address")
+      .style(boldStyle);
+    sheet.cell(4, 2).string(collateralAddress);
+
+    // Financial contracts list
+    sheet
+      .cell(2, 7)
+      .string(`Financial Contracts that use ${collateralData.collateralSymbol}`)
+      .style({ font: { size: 14, bold: true } });
+    sheet
+      .cell(3, 7)
+      .string("Contract Address")
+      .style(boldStyle);
+    sheet.column(7).setWidth("Contract Address".length);
+    sheet
+      .cell(3, 8)
```

Comment:
THese hard coded coordinates (3,8) seem like they will be nightmare to maintain and update if we want to change the format of the sheet. Thoughts on declaring these as constants and then just adding offsets?

For example:
```
COL_COLLATERAL_SYMBOL = 1
COL_COLLATERAL_TVL = 2
```

Reply:
ye, I agree. it's not ideal. it's bound very specifically to the design of this sheet in particular. not really any better way of making it more generalised giving it's building up on a per cell basis.